config/routes.rb

```diff
@@ -4,7 +4,8 @@
     namespace :v0 do
       resources :pings, only: [:index], constraints: { format: 'json' }
     end
-
+    
+    resources :orders, only: [:create, :update] 
```

Comment:
I think you should have create a show action for the order as well, so that when the user wanted to view the order you would have responded with even more detailed information about the order than you got with the response from create and update. For example a total price and detailed data like that